agendas/2024/11-Nov/07-wg-primary.md

```diff
@@ -126,3 +127,8 @@ hold additional secondary meetings later in the month.
    - [All open action items (by meeting)](https://github.com/graphql/graphql-wg/projects?query=is%3Aopen+sort%3Aname-asc)
 1. Make `reason` non-nullable in `@deprecated` (@martinbonnin, 5 min)
    - https://github.com/graphql/graphql-spec/pull/1040
+1. Discuss enhancing deprecation (@gwardwell, 10 min)
```

Comment:
Going to allocate a little more time for this, we'll likely get into the weeds on some of these points!

```suggestion
1. Discuss enhancing deprecation (@gwardwell, 15 min)
```